source/io/module_javaScriptInvoke.js

```diff
@@ -68,7 +68,9 @@
         // Disable new-cap for the cwrap functions so the names can be the same in C and JS
         /* eslint 'new-cap': ['error', {'capIsNewExceptions': [
             'JavaScriptInvoke_GetParameterPointer',
-            'JavaScriptInvoke_GetParameterType',
+            'JavaScriptInvoke_GetParameterTypeName',
```

Comment:
JavaScriptInvoke_GetParameterTypeName is no longer needed correct? we should remove it.

Reply:
And like Carlos mentioned we can get rid of JavaScriptInvoke_GetParameterPointer and we can probably also get rid of JavaScriptInvoke_GetArrayElementType

Reply:
Good point. Will do.